crates/sncast/src/starknet_commands/verification/base.rs

```diff
@@ -0,0 +1,90 @@
+use anyhow::{anyhow, Context, Result};
+use async_trait::async_trait;
+use camino::Utf8PathBuf;
+use reqwest::StatusCode;
+use serde::Serialize;
+use sncast::response::structs::VerifyResponse;
+use sncast::Network;
+use starknet::core::types::FieldElement;
+use std::ffi::OsStr;
+use walkdir::WalkDir;
+
+#[async_trait]
+pub trait VerificationInterface {
+    fn new(network: Network, workspace_dir: Utf8PathBuf) -> Self;
+    async fn verify(
```

Comment:
Both implementations of this method are identical. You can make it a default implementation